src/vehicle.cpp

```diff
@@ -875,6 +875,23 @@ void vehicle::use_controls()
         menu.addentry( release_remote_control, true, 'l', _("Stop controlling") );
     }
 
+    bool has_electronic_controls = false;
+
+    if ( remotely_controlled ) {
+        for( size_t p = 0; p < parts.size(); ++p ) {
+            if (part_flag(p, "CTRL_ELECTRONIC")) {
+                has_electronic_controls = true;
+            }
+        }
+    } else {
+        //TODO: Global coords have to be converted to relative position before it will work
+        std::vector<int> parts_for_check = parts_at_relative( pos_global.x, pos_global.y);
```

Comment:
I tried to write the correct code with assumption that you can install a few parts with different control flags at one tile. ```global_part_at``` returns only one id of the installed part, but I want to iterate over all parts in the tile.